File: server/service/src/main/java/by/iba/dto/resp/user/UserResp.java
package by.iba.dto.resp.user;

import by.iba.common.dto.core.FullAbstractResp;
import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserResp extends FullAbstractResp {

    private String firstName;
    private String lastName;
    private String email;
    private List<UserRoleDTO> roles;
    private String imageUrl;
    @JsonIgnore
    private String password;

}

File: client/app/src/main/java/by/chekun/repository/database/entity/user/UserResp.kt
package by.chekun.repository.database.entity.user

import java.time.LocalDateTime

class UserResp {

    val version: Int? = null
    val dateOfCreation: LocalDateTime? = null
    val dateOfLastUpdate: LocalDateTime? = null
    val id: Long? = null
    val firstName: String? = null
    val lastName: String? = null
    val email: String? = null
    val roles: List<UserRoleDTO>? = null
    val imageUrl: String? = null
    val password: String? = null
}